### PR TITLE
Add bundling for synthetic fabric

### PR DESCRIPTION
**Pull request by pjf**
#### Summary

Balance "Add bundling for synethic fabric"

#### Purpose of change

Synthetic patches could not be bundled, making them extremely slow to move in bulk. This adds bundling for them, just like we have for leather, cotton, etc.

#### Describe the solution

JSON edits.

#### Describe alternatives you've considered

Using bundles of 100 rather than 50.

#### Testing

Applied changes to my current game. Bundled some synthetic fabric. Took it back to base. Unbundled it.

#### Additional context

None

### CODE REVIEW

**Review by Maleclypse — APPROVED** [review]

**Review by Maleclypse — CHANGES_REQUESTED** [review]
```/tests/item_test.cpp:739: FAILED:
(~[slow] ~[.],starting_items)=>   CHECK( item_density( target ) <= max_density )
(~[slow] ~[.],starting_items)=> with expansion:
Error: (~[slow] ~[.],starting_items)=>   1.152f <= 1.15f
(~[slow] ~[.],starting_items)=> with messages:
(~[slow] ~[.],starting_items)=>   target.typeId() := string_id( "bundle_nylon" )
(~[slow] ~[.],starting_items)=>   target.weight() := 288000mg
(~[slow] ~[.],starting_items)=>   target.volume() := 250ml
```

I think you've made it too dense  per the test or maybe not dense enough I'm not good with this particular test.

File: data/json/items/generic.json [hunk 1]
```diff
@@ -488,6 +488,22 @@
...
     "color": "white"
   },
+  {
+    "id": "bundle_nylon",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": { "str": "bundle of synthetic fabric", "str_pl": "bundles of synthetic fabric" },
+    "description": "Fifty synthetic fabric patches, bundled tightly together for storage.  Disassemble to unpack.",
+    "//": "Weight is 50 x 5750mg per patch",
+    "weight": "288 g",
```
Comment (Karol1223) [inline data/json/items/generic.json:498]: That equals to 287.5g, not 288, making your density test fail. Do that in milligrams and it'll be fine.

File: data/json/recipes/recipe_deconstruction.json [hunk 2]
```diff
@@ -5092,6 +5092,15 @@
...
   },
+  {
+    "result": "bundle_nylon",
+    "type": "uncraft",
+    "activity_level": "NO_EXERCISE",
+    "skill_used": "fabrication",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "nylon", 50 ] ], [ [ "thread", 8 ] ] ],
+    "flags": [ "BLIND_EASY" ]
+  },
```
Comment (Karol1223) [inline data/json/recipes/recipe_deconstruction.json:5103]: Don't do that. Make the recipe reversible. If you give it a dedicated uncraft like this for some reason people will be able to transform sinew/plant fiber/yarn into thread.

**Review by Karol1223 — CHANGES_REQUESTED** [review]

File: data/json/items/generic.json (hunk 1 above)
Comment (pjf) [reply data/json/items/generic.json]: "Nobody will notice if I round this by half-a-gram, just for the thread..."  :)

Updated commit uploaded. Fingers crossed the density test will smile upon me.

File: data/json/recipes/recipe_deconstruction.json (hunk 2 above)
Comment (pjf) [reply data/json/recipes/recipe_deconstruction.json:5103]: Oh dang. All the other bundles have dedicated uncrafts, so I just went with consistency.

Making it reversible is much better. Fixing now.

Comment (Karol1223) [reply data/json/recipes/recipe_deconstruction.json]: Yeah old bundles started ages ago and then everyone just copied the same method. But reversible is the better solution - you can even make it take less than tying it together (I've been doing it for reversible recipes for some bags)

**Comment by pjf** [comment]
Density tests look like they're now passing. I also crafted and uncrafted a bundle in-game to verify it's working with changes, so I think this is good to go! :tada:

**Review by Maleclypse — APPROVED** [review]
Thanks!

**Pull request merged**